tests/suites/0_stateless/20+_others/20_0001_planner_v2.sql

```diff
@@ -178,6 +178,13 @@ insert into temp values (1);
 select a from temp;
 drop table temp;
 
+
+-- CASE WHEN
+select '=== Test CASE-WHEN ===';
+select count_if(a = '1'), count_if(a = '2'), count_if(a = '3'), count_if(a is null) from (
+	SELECT (CASE WHEN number % 4 = 1 THEN '1' WHEN number % 4 = 2 THEN '2' WHEN number % 4 = 3 THEN '3' END) as a FROM numbers(100)
+);
```

Comment:
Maybe we can add some test cases for `ELSE`?

For example:
```sql
select case 1 when 2 then 3 else 4 end;
```

Reply:
By the way, we should add some test cases for the situation that data types of braches are different(numeric, string, nullability) as well.